packages/zoe/src/contracts/loan/scheduleLiquidation.js

```diff
@@ -17,12 +18,20 @@ export const scheduleLiquidation = (zcf, configWithBorrower) => {
     mmr,
   } = configWithBorrower;
 
+  // TODO(hibbert) drop mmr as Percent before Beta
+  let {
+    mmrRatio, // Maintenance Margin Requirement, as a ratio
+  } = configWithBorrower;
+  if (!mmrRatio) {
+    mmrRatio = mmr.makeRatio();
+  }
+
```

Comment:
This is internal, so it doesn't make sense to pass around mmr. We should already have mmrRatio.